packages/nimble-blazor/Tests/SprightBlazor.Tests.Acceptance/WebHostServerFixture.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+using Xunit;
+
+namespace SprightBlazor.Tests.Acceptance;
+
+public abstract class WebHostServerFixture : IAsyncLifetime, IDisposable
```

Comment:
There is enough implementation between this class and `PlaywrightFixture` (maybe others?) where it would be nice if we could leverage a single implementation between the Acceptance Test projects in NimbleBlazor and SprightBlazor. This likely calls for the creation of another project.

Reply:
Changed this PR to only partially address the Blazor feature and captured the remaining tasks there: https://github.com/ni/nimble/issues/1976
@m-akinc going forward lets continue to do that where it's appropriate to split work in a different PR and make sure it's tracked for the goal of smaller more targeted PRs.